test/metabase/query_processor_test/expression_aggregations_test.clj

```diff
@@ -269,8 +269,36 @@
   (mt/test-drivers (mt/normal-drivers-with-feature :expression-aggregations)
     (testing "can we use named expressions inside expression aggregations?"
       (is (= [[406]]
-             (mt/format-rows-by [int]
-               (mt/rows
-                 (mt/run-mbql-query venues
-                   {:aggregation [[:sum [:expression "double-price"]]]
-                    :expressions {"double-price" [:* $price 2]}}))))))))
+             (mt/formatted-rows [int]
```

Comment:
Does `mt/formatted-rows` automatically "extract" only rows from a query? Is that the reason we removed `mt/rows` in the previous step?

Reply:
`mt/formatted-rows` is the same as `mt/rows` + `mt/format-rows-by` so I just consolidated them